docs/telemetry.md

```diff
@@ -25,3 +29,25 @@ The collected data looks like this.
   "type": "track"
 }
 ```
+
+## **What We Collect**
+
+Analytics tracked include:
+- The unique id generated from the MAC address.
+- The testkube version.
+- The command being executed.
```

Comment:
Maybe we could point out that we don't track any custom parameters and flags passed to a command by user - only top level testkube commands for general stats ?

Reply:
@jfermi ^^

Reply:
@exu, in the details field  we are tracking the whole command

Reply:
So it sounds like we can't say "we don't track customer parameters and flags passed" right now?